### ISSUE TITLE
`names` generator broken on `Owners` object

### ISSUE THREAD

**Issue by clokep** [ThreatConnect-Inc/threatconnect-python#1]
The [generator on the `Owners` object to get a iterator of the `name` of each `Owner`](https://github.com/ThreatConnect-Inc/threatconnect-python/blob/cedef1e9f69dd53787531d37b49cb38ec95a58b6/threatconnect/Resources/Owners.py#L57-L61) has a minor bug in it. `get_name` is not a property on the `Owner` class, I believe this should be simply `name`.

```diff
     @property
     def names(self):
         """ generator for owner names """
         for obj in self._objects:
-            yield obj.get_name()
+            yield obj.name
```

I can try to find time to put up a PR, if you'd prefer.

**Comment by cblades** [comment]
Thanks for finding and reporting.  This issue is fixed in master, see [29716d6](https://github.com/ThreatConnect-Inc/threatconnect-python/commit/29716d6909945ef0203463ec2a86ec3d3a19cf6d)

**Comment by clokep** [comment]
@cblades Thanks for fixing this so quickly!
